WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -145,6 +171,11 @@ inline static BOOL valuesAreEqual(id value1, id value2)
   return value1 == value2 || [value1 isEqual:value2];
 }
 
+inline static BOOL valuesAreEqualOrBlank(id value1, id value2)
```

Comment:
I'd rather start the method name with `areValues...` to align with the other functions style